routes/api.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const router = express.Router();
+
+const db = require("../db/models");
```

Comment:
It looks like you are creating a new variable that contains the Review model. Instead you can destructure the view like so `const { Review } = require("../db/models");`

Reply:
Fixed!